docs/enterprise/metrics-app-overview-page.md

```diff
@@ -30,28 +30,15 @@ The Multi metric column ribbon displays the four values listed below. It shows t
 
 * **Users** - The number of users that performed operations.
 
-## CU over time
+## Capacity utilization and throttling
 
-Displays the CU usage of the selected capacity over time. Use the tabs at the top of the visual to toggle how the visual is displayed:
+It has three features. Use the tabs at the top of the visuals to toggle.
 
-* **Linear** - Display the information using a linear scale that starts at 0 percent.
-
-* **Logarithmic** - Display the information using a logarithmic scale that depends on your CUs consumption.
```

Comment:
This view is useful for low utilization scenarios or scenarios where large spikes in usage have occured e.g. when a capacity has been paused.

Reply:
We are having this description of view in below.